bin/index.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 const inspect = require('../dist/index')
 
-inspect.buildDepTreeFromFiles('./', 'package.json', 'package-lock.json')
+inspect.buildDepTreeFromFiles('./', 'package.json', 'package-lock.json', 'yarn.lock')
```

Comment:
Hi, I don't believe this code works as you expect. 

To illustrate, here is the function signature for this function 
```typescript
function buildDepTreeFromFiles(
    root: string, 
    manifestFilePath: string, 
    lockFilePath: string, 
    includeDev?: boolean, 
    strictOutOfSync?: boolean
): Promise<PkgTree>
```

This would mean we are passing `yarn.lock` to the `includeDev` argument. 

I would suggest updating so that the 3rd argument - `lockFilePath` - is conditionally passed as `package-lock.json` or `yarn.lock` based on some trigger. Maybe an cli flag as this is a "bin".